packages/ui/src/components/Designer/index.tsx

```diff
@@ -206,15 +262,17 @@ Check this document: https://pdfme.com/docs/custom-schemas`);
         }}
         zoomLevel={zoomLevel}
         setZoomLevel={setZoomLevel}
+        addPageAfter={isBlankPdf(template.basePdf) ? handleAddPageAfter : undefined}
+        removePage={isBlankPdf(template.basePdf) ? handleRemovePage : undefined}
```

Comment:
is `isBlankPdf` efficient? should we store the result to avoid the additional call?